src/primenodes.cpp

```diff
@@ -12,6 +12,9 @@ extern void CloseDb(const string& strFile);
 
 CPrimeNodeDB* primeNodeDB;
 
+// Reset all primenode stakerates to 100% after the given date
+static const unsigned int RESET_PRIMERATES = 1429531200; // Mon, 20 Apr 2015 12:00:00 GMT
```

Comment:
This isn't used anywhere outside of primenodes.cpp so don't declare it in header at all.